src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImpl.java

```diff
@@ -290,10 +290,12 @@ public SubmissionApiDelegateImpl(
 
         GetSubmissionConfigResponse response = new GetSubmissionConfigResponse();
 
-        response.setClientApplication(fromCacheSubmission.get().getClientApplication());
+        response.setClientAppName(fromCacheSubmission.get().getClientApplication().getDisplayName());
```

Comment:
question: do we still need the clientApplication object with a display name property or can we just have a value for clientAppName?

Reply:
yes we do lol